go/cmd/dolt/commands/sqlserver/sqlclient.go

```diff
@@ -481,11 +482,14 @@ func BuildConnectionStringQueryist(ctx context.Context, cwdFS filesys.Filesys, c
 		return nil, err
 	}
 
-	parsedMySQLConfig, err := mysqlDriver.ParseDSN(ConnectionString(clientConfig, database))
+	dbName, _ := dsess.SplitRevisionDbName(database)
+	parsedMySQLConfig, err := mysqlDriver.ParseDSN(ConnectionString(clientConfig, dbName))
 	if err != nil {
 		return nil, err
 	}
 
+	// ParseDSN currently doesn't support `/` in the db name
```

Comment:
This comment seems like it's in the wrong place. move it up.

Also, "database" is actually a databaseRevision (maybe dbRev?). Renaming it will help future readers.